cli/cmd/plugin/unmanaged-cluster/tanzu/tanzu.go

```diff
@@ -134,6 +134,11 @@ func (t *UnmanagedCluster) Deploy(scConfig *config.UnmanagedClusterConfig) error
 	log.AddLogFile(bootstrapLogsFp)
 	log.Event(logger.FolderEmoji, "Created cluster directory")
 
+	// Log a warning if the user has given a ProviderConfiguration
+	if scConfig.ProviderConfiguration != "" {
+		log.Style(outputIndent, color.FgYellow).ReplaceLinef("Reading ProviderConfiguration from config file. All other provider specific configs may be ignored")
```

Comment:
OK, I feel like a jerk posting this nit, but since you start with a full sentence that ends with a period, can you end the second sentence with a period?

Sorry. :]

Reply:
Grammar! My greatest enemy! 😂 happy to fix!